.vscode/launch.json

```diff
@@ -9,7 +9,41 @@
       "type": "python",
       "request": "launch",
       "module": "homeassistant",
-      "args": ["--debug", "-c", "config"]
+      "args": [
+        "--debug",
+        "-c",
+        "config"
+      ]
+    },
+    {
+      // Debug by attaching to local Home Asistant server using Remote Python Debugger.
+      // See https://www.home-assistant.io/integrations/debugpy/
+      "name": "Debug Home Assistant: Attach Local",
```

Comment:
While I get the copy from the example, the name is a bit confusing with the one above.
The main one is now `Home Assistant`, while these start with `Debug Home Assistant`. 

It is confusing in a sense, that one might wonder if the first isn't debuggable? (which it is).

So, maybe remove the `Debug ` from this one and the one below.

Reply:
Yes you are right. And do you think we should set the first one with "Home Assistant: Launch" ?

Reply:
It is the launcher already :)

I think it is fine like this 👍